packages/graphql-contentful-core/README.md

```diff
@@ -175,6 +175,8 @@ const mappers = {
 
 PathsConfigs allow you to define how a path is constructed from a specific content type. There are two ways to define these:
 
+**Important:** When using Sanity as the CMS, path resolution is not currently supported. Any queries relying on path readers will return no results.
```

Comment:
Not sure why this is necessary. Path resolution is done using PathConfig in the extensions and should be able to be supported without much effort in Sanity

Reply:
I think it did add a change where it returns null